computer_monitoring/src/log_ram_temp.py

```diff
@@ -84,6 +82,6 @@ def collect_and_publish(self):
             self.rate.sleep()
     
 if __name__ == "__main__":
-    rospy.init_node("pi_monitor_node")
+    rospy.init_node("pi_monitor_node", anonymous=True)
```

Comment:
can you change the node name? computer_monitor_node or something

Reply:
I see. but it worth changing the name inside the node to a more general name similar to the package name.
The code looks good. 
I will merge.